python/ray/_private/ray_logging.py

```diff
@@ -102,15 +102,13 @@ def run_callback_on_events_in_ipython(event: str, cb: Callable):
     Args:
         cb: The callback to run.
     """
-    try:
+    if "IPython" in sys.modules:
```

Comment:
This is currently how other places (e.g. in widgets) checks if ipython is available. It doesn't do the try import...